views/applications/all.html

```diff
@@ -0,0 +1,49 @@
+{% extends "_donut.html" %}
+
+{% block pageHead %}
+  {{ super() }}
+{% endblock %}
+
+{% block header %}
+  {{ super() }}
+{% endblock %}
+
+{% block main %}
+  <div class="introduction">
+    <h1 class="introduction--h1">{{structure.name}}</h1>
+    <p class="introduction--p">{{structure.description}}</p>
+    <a href="/${config.base}/${config.actions.add}" class="introduction--button">{{config.actions.add|capitalize}} {{config.base}}</a>
```

Comment:
`here` in the template or `here` in this PR's comments?

As for this particular line, it follows the [structure already established for content](https://github.com/punchcard-cms/punchcard/blob/master/views/content/landing.html#L14)